cmd/fiovb.c

```diff
@@ -45,7 +45,7 @@ int do_fiovb_read_pvalue(cmd_tbl_t *cmdtp, int flag, int argc,
 	size_t bytes_read;
 	void *buffer;
 	char *endp;
-	char fiovb_name[30] = { 0 }; /* fiovb.name */
+	char fiovb_name[40] = { 0 }; /* fiovb.name */
```

Comment:
@ricardosalveti @ldts There's no issue with making this larger right?

Reply:
@vanmaegima we should also do the same for other places:

```
u-boot$ grep -ne "fiovb_name\[" -r ./
./cmd/fiovb.c:49:	char fiovb_name[30] = { 0 }; /* fiovb.name */
./cmd/fiovb.c:93:	char fiovb_name[30] = { 0 }; /* fiovb.name */
./cmd/fiovb.c:124:	char fiovb_name[30] = { 0 }; /* fiovb.name */

```